workflows/rnaseq_expression_classification/rnaseq_expression_classification_from_bams.wdl

```diff
@@ -109,7 +109,8 @@ workflow rnaseq_expression_classification_from_bams {
 
     parameter_meta {
         in_bams: {
-            help: "Provide bams to run for comparison"
+            help: "Provide bams to run for comparison",
+            patterns: ["*.bam"]
```

Comment:
Think using `patterns` is going to make the `wdldoc` section ugly. Don't need to change it on your end, I can add handling for it in a new release of `wdldoc`. Just in the meantime, won't look great in the docs.